index.js

```diff
@@ -292,6 +292,72 @@ Bot.prototype.postTo = function(name, text, params, cb) {
 };
 
 /**
+ * Posts a reaction (emoji) to a message by timestamp
+ * @param {string} id - channel ID
+ * @param {string} emoji - emoji string (without the : symbols)
+ * @param {string} ts - timestamp of the message you want to react to
+ * @param {object} params
+ * @returns {vow.Promise}
+ */
+Bot.prototype.postReactionToChannel = function(id, emoji, ts, params) {
+    params = extend({
+        channel: id,
+        name: emoji,
+        timestamp: ts
+    }, params || {});
+
+    return this._api('reactions.add', params);
+};
+
```

Comment:
Hm.. are you passing the timestamp parameter?   Could you post some sample code?